.github/workflows/push-trigger.yml

```
@@ -362,7 +362,7 @@ jobs:
       matrix:
         include:
           - SERVICE_LOCATION: 'api-test'
-            SERVICE_NAME: 'apitest-esignet'
+            SERVICE_NAME: ${{ matrix.SERVICE_NAME }}'
```

actionlint (1.7.9)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)